src/pages/sighting/SightingCore.jsx

```diff
@@ -0,0 +1,279 @@
+import EntityHeaderNew from '../../components/EntityHeaderNew';
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import useDeleteSighting from '../../models/sighting/useDeleteSighting';
+import useCommitAssetGroupSighting from '../../models/assetGroupSighting/useCommitAssetGroupSighting';
+import useSightingFieldSchemas from '../../models/sighting/useSightingFieldSchemas';
+import { formatDate } from '../../utils/formatters';
+// import AnnotationsGallery from './AnnotationsGallery';
+// import IndividualsGallery from './IndividualsGallery';
+import Annotations from './Annotations';
+import Photographs from './Photographs';
+import OverviewContent from './OverviewContent';
+import SightingHistoryDialog from './SightingHistoryDialog';
+import FeaturedPhoto from './featuredPhoto/FeaturedPhoto';
+import Encounters from './encounters/Encounters';
+
+export default function SightingCore({
+  data,
+  loading,
+  error,
+  statusCode,
+  pending,
+  id,
+}) {
+  const history = useHistory();
+  const intl = useIntl();
+  const queryClient = useQueryClient();
+
+  const fieldSchemas = useSightingFieldSchemas();
+
+  function refreshData() {
+    const queryKey = pending
+      ? getAGSQueryKey(id)
+      : getSightingQueryKey(id);
+    queryClient.invalidateQueries(queryKey);
+  }
+
+  const {
+    deleteSighting,
+    loading: deleteInProgress,
+    error: deleteSightingError,
+    setError: setDeleteSightingError,
+  } = useDeleteSighting();
+
+  const {
+    commitAgs,
+    isLoading: commitAgsLoading,
+  } = useCommitAssetGroupSighting();
+
+  /*
+    known issue: if data or fieldschemas change values
+    or properties this may not update
+    switch to data.version?
+  */
+  const metadata = useMemo(
+    () => {
+      if (!data || !fieldSchemas) return null;
+      return fieldSchemas.map(schema => ({
+        ...schema,
+        value: schema.getValue(schema, data),
+      }));
+    },
+    [data, fieldSchemas],
+  );
+
+  // fetch data for Id...
+  useDocumentTitle(`Sighting ${id}`, { translateMessage: false });
+
+  const [historyOpen, setHistoryOpen] = useState(false);
+  const [deleteDialogOpen, setDeleteDialogOpen] = useState(false);
+
+  const activeTab = window.location.hash || '#overview';
+
+  if (loading) return <LoadingScreen />;
+  if (statusCode === 404)
+    return (
+      <SadScreen
+        subtitleId="SIGHTING_NOT_FOUND"
+        descriptionId="SIGHTING_NOT_FOUND_DESCRIPTION"
+        variant="genericError"
+      />
+    );
+  if (error) return <SadScreen variant="genericError" />;
+  if (!data)
+    return (
+      <SadScreen
+        variant="notFoundOcean"
+        subtitleId="SIGHTING_NOT_FOUND"
+      />
+    );
+
+  const assets = get(data, 'assets', []);
+  const annotations = flatten(
+    assets.map(asset => get(asset, 'annotations')),
+  );
+  const allAnnotationsAssigned = every(annotations, 'encounter_guid');
+  const detectionComplete = get(data, 'stage') !== 'detection';
```

Comment:
This still feels brittle to me. Disregard if need be; I think that you had a good reason for keeping this the last time that I saw it.

Reply:
Detection is the first stage, so if it's in any other stage it's through detection